db/options.go

```diff
@@ -149,3 +149,18 @@ func WithManagedDBToken(t thread.Token) ManagedDBOption {
 		args.Token = t
 	}
 }
+
+// InviteInfoOptions defines options getting DB invite info.
+type InviteInfoOptions struct {
+	Token thread.Token
+}
+
+// InviteInfoOption specifies a managed db option.
+type InviteInfoOption func(*InviteInfoOptions)
+
+// WithInviteInfoToken provides authorization for accessing DB invite info.
+func WithInviteInfoToken(t thread.Token) InviteInfoOption {
+	return func(args *InviteInfoOptions) {
+		args.Token = t
+	}
+}
```

Comment:
Above are new options for fetching db invite info. This is basically just a proxy for the thread options that are needed to call GetThread in the net layer.